test/gtest/uct/test_uct_perf.cc

```diff
@@ -131,8 +131,7 @@ const test_perf::test_spec test_uct_perf::tests[] =
 
 UCS_TEST_P(test_uct_perf, envelope) {
     if (has_transport("cm") ||
-        has_transport("ugni_udt") ||
-        has_transport("cuda_ipc")) {
```

Comment:
this is not exactly needed, because cuda_ipc is already excluded below when instating tests:
`UCT_INSTANTIATE_NO_SELF_TEST_CASE(test_uct_perf);`
it doesn't contain self and CUDA TLs

Reply:
@dmitrygx This means that `cuda_ipc` UCT won't be skipped correct?

Reply:
> This means that `cuda_ipc` UCT won't be skipped correct?

@Akshay-Venkatesh we are already skipping `cuda_ipc` UCT here since test_uct_perf are instantiated by `UCT_INSTANTIATE_NO_SELF_TEST_CASE(test_uct_perf);` that doens't define CUDA TLs